tests/testthat/test-Rules-methods.R

```diff
@@ -4361,10 +4361,18 @@ test_that("tidy-IncrementsRelativeParts works correctly", {
   expect_snapshot_value(result, style = "deparse")
 })
 
+# Relevant:https://github.com/Roche/crmPack/issues/759
 test_that("tidy-NextBestNCRM works correctly", {
   obj <- .DefaultNextBestNCRM()
   result <- tidy(obj)
-  expect_snapshot_value(result, style = "deparse")
+  expected <- tibble::tibble(
```

Comment:
ah yeah, so this is a good example where snapshot testing deceived us into thinking everything is fine

Reply:
... but also partly my fault for not checking the original snapshot to notice that its value was incorrect.  Subsequent snapshots passed because the incorrect value didn't change.  Human eyeball Mk 1 picked up the problem as soon as it appeared in real life.